commands/transfer_funds.go

```diff
@@ -144,3 +123,129 @@ func checkTransfer(cmd *commandspb.Transfer) (e Errors) {
+		} else if nTopPerformers.GreaterThan(num.DecimalOne()) {
+			errs.AddForProperty(prefix+".n_top_performers", ErrMustBeBetween01)
+		}
+	}
+	if dispatchStrategy.EntityScope == vega.EntityScope_ENTITY_SCOPE_INDIVIDUALS && dispatchStrategy.IndividualScope == vega.IndividualScope_INDIVIDUAL_SCOPE_UNSPECIFIED {
+		errs.AddForProperty(prefix+".individual_scope", ErrIsRequired)
+	}
+	if dispatchStrategy.EntityScope != vega.EntityScope_ENTITY_SCOPE_INDIVIDUALS && dispatchStrategy.IndividualScope != vega.IndividualScope_INDIVIDUAL_SCOPE_UNSPECIFIED {
+		errs.AddForProperty(prefix+".individual_scope", ErrIsNotValid)
+	}
+	if dispatchStrategy.DistributionStrategy == vega.DistributionStrategy_DISTRIBUTION_STRATEGY_UNSPECIFIED && toAccountType != vega.AccountType_ACCOUNT_TYPE_REWARD_MARKET_PROPOSERS {
+		errs.AddForProperty(prefix+".distribution_strategy", ErrIsRequired)
+	}
+	if dispatchStrategy.DistributionStrategy != vega.DistributionStrategy_DISTRIBUTION_STRATEGY_UNSPECIFIED && toAccountType == vega.AccountType_ACCOUNT_TYPE_REWARD_MARKET_PROPOSERS {
+		errs.AddForProperty(prefix+".distribution_strategy", ErrIsNotValid)
+	}
+	if len(dispatchStrategy.StakingRequirement) > 0 {
+		if toAccountType == vega.AccountType_ACCOUNT_TYPE_REWARD_VALIDATOR_RANKING || toAccountType == vega.AccountType_ACCOUNT_TYPE_REWARD_MARKET_PROPOSERS {
+			errs.AddForProperty(prefix+".staking_requirement", ErrIsNotValid)
+		} else if staking, ok := big.NewInt(0).SetString(dispatchStrategy.StakingRequirement, 10); !ok {
+			errs.AddForProperty(prefix+".staking_requirement", ErrNotAValidInteger)
+		} else if staking.Cmp(big.NewInt(0)) < 0 {
+			errs.AddForProperty(prefix+".staking_requirement", ErrMustBePositiveOrZero)
+		}
+	}
+	if len(dispatchStrategy.NotionalTimeWeightedAveragePositionRequirement) > 0 {
+		if toAccountType == vega.AccountType_ACCOUNT_TYPE_REWARD_VALIDATOR_RANKING || toAccountType == vega.AccountType_ACCOUNT_TYPE_REWARD_MARKET_PROPOSERS {
+			errs.AddForProperty(prefix+".notional_time_weighted_average_position_requirement", ErrIsNotValid)
+		} else if notional, ok := big.NewInt(0).SetString(dispatchStrategy.NotionalTimeWeightedAveragePositionRequirement, 10); !ok {
+			errs.AddForProperty(prefix+".notional_time_weighted_average_position_requirement", ErrNotAValidInteger)
+		} else if notional.Cmp(big.NewInt(0)) < 0 {
+			errs.AddForProperty(prefix+".notional_time_weighted_average_position_requirement", ErrMustBePositiveOrZero)
+		}
+	}
+	if dispatchStrategy.WindowLength > 0 && toAccountType == vega.AccountType_ACCOUNT_TYPE_REWARD_MARKET_PROPOSERS {
+		errs.AddForProperty(prefix+".window_length", ErrIsNotValid)
+	}
+	if dispatchStrategy.WindowLength > 100 {
+		errs.AddForProperty(prefix+".window_length", ErrMustBeAtMost100)
+	}
+	if len(dispatchStrategy.RankTable) == 0 && dispatchStrategy.DistributionStrategy == vega.DistributionStrategy_DISTRIBUTION_STRATEGY_RANK {
+		errs.AddForProperty(prefix+".rank_table", ErrIsNotValid)
```

Comment:
I think we have this one:
```suggestion
		errs.AddForProperty(prefix+".rank_table", ErrMustBePositive)
```

... to say, must be more than 0